app/components/PageTitle.js

```diff
@@ -1,15 +1,23 @@
 // @flow
 import * as React from "react";
+import { observer, inject } from "mobx-react";
 import { Helmet } from "react-helmet";
+import AuthStore from "stores/AuthStore";
 
 type Props = {
   title: string,
   favicon?: string,
+  auth: AuthStore,
 };
 
-const PageTitle = ({ title, favicon }: Props) => (
+const { auth } = this.props;
+const { team } = auth;
```

Comment:
This doesn't make sense, you can't access props outside the scope of the component

```suggestion
```